src/forms-opportunities/utils.js

```diff
@@ -182,9 +182,9 @@ export async function generateOpptyDataForHighPageViewsLowFormNav(formVitals, co
   );
 }
 
-export function isSearchForm(scrapedFormData) {
+export function shouldExcludeForm(scrapedFormData) {
   return scrapedFormData?.formType === 'search'
-    || scrapedFormData?.classList?.includes('search')
+    || scrapedFormData?.formType === 'login'
```

Comment:
I hope you are planning to have the classList check for search forms inside scraper itself